npc/re/warps/cities/izlude.txt

```diff
@@ -72,6 +73,82 @@ izlude_b,52,172,0	warp	iz04_b	2,2,izlude_in,74,161
 izlude_c,52,172,0	warp	iz04_c	2,2,izlude_in,74,161
 izlude_d,52,172,0	warp	iz04_d	2,2,izlude_in,74,161
 
+//= iz_int(Intro) & int_land(Secluded Island) ================
+iz_int,27,30,0	script	noviship#room1-1	WARPNPC,2,2,{
+	end;
+
+OnTouch:
+	warp(strnpcinfo(NPC_MAP), 51, 30);
+	end;
+OnInit:
+	while (true) {
+		emotion e_gasp, 0;
+		sleep 8000;
+	}
+	end;
+}
+
+iz_int,47,30,3	script	noviship#room1-2	WARPNPC,2,2,{
+	end;
+
+OnTouch:
+	@display = 0;
```

Comment:
To clear the variable of wounded swordsman in case player switches map (classchange command). I thought its cleared after the end but it wasn't the case. Works fine if player logs out in iz_int but if switched maps the npc sprite didn't change back.